extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -178,17 +187,23 @@ export class AmpStoryReaction extends AMP.BaseElement {
   }
 
   /** @override */
```

Comment:
Is this override? I'm overriding buildCallback from the concrete classes to call this method with their specific CSS anyways